modules/document_repository/templates/menu_document_repository.tpl

```diff
@@ -249,7 +249,7 @@
                             </label>
                             <div class="col-xs-8">
                                 <select name="parent_id" id="parent_id" class="form-control input-sm">
-                                    <option value=" "> </option>
+                                    <option value="0"></option>
```

Comment:
It looks to me like the problem is that it's a space (`" "`) instead of the empty string (`""`) as in the line below. Wouldn't changing it to have "0" mean that you can no longer have "0" as a category name?

Can you try changing this to `""` instead of `"0"` (without the other changes below) and see if it fixes your issue? If so, it would be a more robust way to fix it.